.eslintrc.json

```diff
@@ -15,6 +15,7 @@
   "rules": {
     "prettier/prettier": ["error", { "endOfLine": "auto" }],
     "dot-notation": "error",
+    "eol-last": 2,
```

Comment:
Oh, we have a dup https://github.com/w3c/respec/pull/2015 (preference is for new contributor ;) )